index.html

```diff
@@ -5,17 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Personal Portfolio</title>
-    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="main.css">
     <style> @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap'); </style>
 </head>
 <body>
-    <header>
+    <header class="header">
```

Comment:
- [ ] As requested by the previous reviewer, kindly place the `section` tag (and its content) outside the `header` tag. In other words, please close the `header` tag before the `section` tag for better semantics 👍